### PR TITLE
Version 12: UI_TRAFFIC_STATS, Import json configurations with no PreShared-Key, allow clients with no privateKey and more.

### PR DESCRIPTION
**Pull request by pheiduck**
**Stable Release: 2024-03-12**
- UI_TRAFFIC_STATS
- Import json configurations with no PreShared-Key
- allow clients with no privateKey
- Reduce Docker image size
- Added Thai language
- Added Italian language. Updated supported languages list
List see: https://github.com/wg-easy/wg-easy/milestone/1?closed=1

Thanks to all contributors to get us able to release v12!
v12 Feature Window is closed now, focus on testing and bug fixes if needed.
Feedback of our nightly users are welcome!

### CODE REVIEW

**Comment by pheiduck** [comment]
Will be merge ready if approval is given by @peterlewis.

**Comment by peterlewis** [comment]
Will be able to review in an hour or so! :)

**Comment by pheiduck** [comment]
No rush, will released on 03-12

File: src/www/css/app.css [hunk 1]
```diff
@@ -768,10 +788,6 @@ video {
   height: 3rem;
 }
 
-.h-14 {
```
Comment (peterlewis) [inline src/www/css/app.css:771]: Can I just check that these are expected deletions?

File: src/www/css/app.css [hunk 2]
```diff
@@ -768,10 +788,6 @@ video {
...
-  height: 3.5rem;
```
Comment (peterlewis) [inline src/www/css/app.css:772]: Can I just check that these are expected deletions?

File: src/www/css/app.css [hunk 3]
```diff
@@ -784,10 +800,6 @@ video {
   height: 0.75rem;
 }
 
-.h-32 {
```
Comment (peterlewis) [inline src/www/css/app.css:787]: Can I just check that these are expected deletions?

File: src/www/css/app.css [hunk 4]
```diff
@@ -1113,10 +1154,6 @@ video {
   padding-bottom: 3rem;
 }
 
-.pb-2 {
```
Comment (peterlewis) [inline src/www/css/app.css:1116]: Can I just check that these are expected deletions?

File: src/www/css/app.css [hunk 5]
```diff
@@ -1532,8 +1584,12 @@ video {
     display: inline-block;
   }
 
-  .md\:flex-row {
```
Comment (peterlewis) [inline src/www/css/app.css:1535]: Can I just check that we deffo want to get rid of `flex-row`?

**Comment by peterlewis** [comment]
Thanks! Have added a few comments, but otherwise looking good... :)

File: src/www/css/app.css (hunk 1 above)
Comment (pheiduck) [reply src/www/css/app.css:771]: Yep this changes where done by @suxscribe and not by CI see:
https://github.com/wg-easy/wg-easy/commit/7d19b748452bbf9832974704219bf48a5d10f521

File: src/www/css/app.css (hunk 2 above)
Comment (pheiduck) [reply src/www/css/app.css:772]: Here as well.

File: src/www/css/app.css (hunk 3 above)
Comment (pheiduck) [reply src/www/css/app.css:787]: Here as well.

File: src/www/css/app.css (hunk 4 above)
Comment (pheiduck) [reply src/www/css/app.css:1116]: Same here.

File: src/www/css/app.css (hunk 5 above)
Comment (pheiduck) [reply src/www/css/app.css:1535]: Yep, haven’t seen any reports of a bug from nightly users until now. :)

**Comment by peterlewis** [comment]
Ah, yes of course! :) Sorry, simply raising feedback and positive comments does not, an approval, make! :) On it!

**Review by peterlewis — APPROVED** [review]

**Pull request merged**